test/fuzz/headers.dict

```diff
@@ -1,4 +1,6 @@
 # Dictionary to populate HeaderValueOptions
-"path"
-"method"
-":content-length"
\ No newline at end of file
+":path"
+":method"
+":content-length"
```

Comment:
content-length and transfer-encoding aren't pseudo headers, so no `:` prefix. The pseudo headers you want are `:path`, `:method`, `:scheme` and `:authority`.